scripts/travis/print_logs.sh

```diff
@@ -7,5 +7,5 @@ for FILE in $LOG_FILES; do
   echo "================================================================================"
   echo " $FILE"
   echo "================================================================================"
-  cat $FILE
+  cat $FILE || true
```

Comment:
Can this `cat` fail?

Reply:
If there are no log files, then `for FILE in $LOG_FILES` uses `$LOGS_DIR/*` literally, which does not exist and thus `cat` fails.